lib/stink_bomb/configuration.rb

```diff
@@ -6,8 +6,31 @@ def initialize(options = {})
       self.message = options.fetch(:message) { default_message }
     end
 
+    def raise=(error_class)
+      case error_class
+      when true
+        add_bomb(RaiseBomb.new)
+      when false
```

Comment:
Not right after, but I could see someone wanting to set it globally to `true` and then somewhere specifically in their codebase wanting to disable it for an individual instance.

```ruby
StinkBomb.raise = false
StinkBomb.create(...)
StinkBomb.raise = true
```

Not a very likely scenario, but I think it would be pretty surprising trying to set it to `false` and it still raises an error.